packages/shared/lib/core/network/actions/addOfficialNodesToClientOptions.ts

```diff
@@ -3,9 +3,9 @@ import { activeProfile } from '@core/profile'
 import { get } from 'svelte/store'
 
 export function addOfficialNodesToClientOptions(): void {
-    const { clientOptions, networkProtocol, networkType } = get(activeProfile)
+    const { clientOptions, network } = get(activeProfile)
```

Comment:
Already mentioned to you in Slack, but one next step would be to put `clientOptions` on the network object